CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 # Dependencies
 #------------------------------------------------------------------------------#
 # find kokkos
-find_package(Kokkos 3.7 REQUIRED)
+find_package(Kokkos 4.1 REQUIRED)
```

Comment:
Need to change the wiki as well (once we merge)

Reply:
I think you should bump the minimum requirement in a separate PR.